components/ai_chat/resources/page/components/feature_button_menu/index.tsx

```diff
@@ -11,6 +11,13 @@ import { getLocale } from '$web-common/locale'
 import getPageHandlerInstance, * as mojom from '../../api/page_handler'
 import DataContext from '../../state/context'
 import styles from './style.module.scss'
+import classnames from '$web-common/classnames'
+
+// TODO(nullhook): share these strings with the settings webui
+const getModelSubtitleLocale = (modelKey: string): string => {
+  const key = modelKey.split('-').map(w => w.charAt(0).toUpperCase() + w.slice(1)).join('')
```

Comment:
Instead of this complicated string split, uppercase and transform, how about a simpler format where we can just concat? For example ``const key = `braveLeoModelSubtitle-${modelKey}` `` / `braveLeoModelSubtitle-chat-expanded`.

Reply:
done